docs/technical_reference/flowsheets/electrodialysis_1stack.rst

```diff
@@ -0,0 +1,81 @@
+
+    * `One-stack electrodialysis with concentrate stream recirculation (feed-and-bleed mode) <https://watertap--1385.org.readthedocs.build/en/1385/technical_reference/flowsheets/electrodialysis_1stack_conc_recirc.html>`_.
+
+Implementation
+--------------
+
+The modeled one-stack ED system is illustrated by Figure 1. The feed solution is split into two fluids through a separator unit, entering the diluate and concentrate channels of the ED stack. On the outlet side of the ED stack, all diluate fluids are collected into the total product water, and the concentrate fluids into the total brine stream. Water recovery in this model is volume-based, i.e., the ratio of product volume to total volume of feed solution. The model simulates the steady state of the ED system.
+
+.. figure:: ../../_static/flowsheets/edfs.png
+    :width: 500
+    :align: center
+
+    Figure 1. Flowsheet diagram: one-stack ED system
+
+Documentation for unit models from WaterTAP: 
+    * `Electrodialysis_1D <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/electrodialysis_1D.html>`_
+Documentation for unit models from IDAES: 
+    * `Feed block <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/feed.html>`_
+    * `Separator <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/separator.html>`_
+    * `Product block <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/product.html>`_
+Documentation for the property model:
+    * `Multi-Component Aqueous Solution (MCAS) Property Package <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/mc_aq_sol.html>`_
+
+Degrees of Freedom
+------------------
+The number of degree of freedom (DOF) is associated with the number of fixed variables (parameters) determined by the purpose of the modeling case. We implemented two modeling cases in the `main()` function: (1) the prediction of desalination outcome (salinity of the product water and saline disposal) and (2) the optimization of key decision variables in system design.  In the first case, DOF is set to zero by fixing all initial conditions of the feed solution fluid and definite ED stack parameters. All fixed values are presented in the section to follow.  In the second case, the values of those chosen to be the decision variables in the optimization are unfixed. The DOF number is therefore the number of decision variables. In this example, the decision variables are
```

Comment:
```suggestion
The number of degrees of freedom (DOF) is associated with the number of fixed variables (parameters) determined by the purpose of the modeling case. We implemented two modeling cases in this flowsheet: (1) the prediction of desalination outcome (salinity of the product water and saline disposal) and (2) the optimization of key decision variables in system design.  In the first case, DOF is set to zero by fixing all initial conditions of the feed solution fluid and definite ED stack parameters. All fixed values are presented in the section to follow.  In the second case, the values of those chosen to be the decision variables in the optimization are unfixed. The DOF number is therefore the number of decision variables. In this example, the decision variables are
```
It might be worth avoiding the use of `main()` since somebody reading the docs may not be familiar with what main() entails and I'm not sure if we should expect the reader to have the flowsheet code pulled up.